docs/content/how-to/disconnected/disconnected-workarounds.md

```diff
@@ -0,0 +1,78 @@
+# Disconnected workarounds
+
+## Make ImageContentSourcePolicies work using image tags
+The ImageContentSourcePolicies (ICSP) have not exposed the API to handle the parameter `mirror-by-digest-only`, so in order to do that, we need to manually create a MachineConfig change to be applied in all the HostedCluster workers. This change will perform a similar action in the workers like an ICSP.
+
+!!! note
+
+    You don't need to do this with future versions of Openshift because `config.openshift.io/v1` has an exposed API to do this, called `ImageTagMirrorSet`.
```

Comment:
Should we note what future versions won't need this change? Is it 4.13 or greater?

Reply:
I think the deprecation warning was on 4.13, so I think ICSP could stay here until 4.15 maybe (?). btw does not hurt to keep it here for a while, in the end doc is easy to modify

Reply:
Ok sounds good. I think it's until 4.17.